src/cf-forms/src/molecules/form-fields.less

```diff
@@ -11,15 +12,19 @@
     &__checkbox,
     &__radio {
         .a-label {
+            display: table-row;
```

Comment:
No it's not necessary unless defining a specific structure that requires it. The table display classes will create what they need.